src/ops/ops.go

```diff
@@ -59,9 +59,9 @@ type ops struct {
 }
 
 // NewOps return a new ops interface
-func NewOps(logger *logrus.Logger) Ops {
+func NewOps(logger *logrus.Logger, proxySet bool) Ops {
 	cmdEnv := os.Environ()
-	if config.GlobalConfig.HTTPProxy != "" || config.GlobalConfig.HTTPSProxy != "" {
+	if proxySet && (config.GlobalConfig.HTTPProxy != "" || config.GlobalConfig.HTTPSProxy != "") {
```

Comment:
we must not see config.GlobalConfig.HTTPProxy in controller right? so why we need proxySet ?

Reply:
since this code is used both by the installer and the controller  - we need to differentiate who invokes it.

for the controller - we are using the cluster-wide-proxy - by SetProxyEnvVars in its main. Then we don't need to set it any more, and mainly not to rely on env-vars for proxy other than the cluster ones.

the installer however, requires them in order to wrap the calls to 'nsenter' that doesn't inherit it from the env. This issue (nsenter fails to use env-vars) was detected only on d/s base image.